### ISSUE TITLE
Add aria-label to <article> tags throughout the theme

### ISSUE THREAD

**Issue by cjg89** [UCF/Today-Child-Theme#80]
**Description**

Multiple `article` tags can exist on any given page or post at a time, which require unique names to help differentiate them from other similar landmarks.  Adding `aria-label` or `aria-labelledby` to `article` tags in post templates and feature markup can help satisfy this requirement.

**Why it's Important**

Users may not understand the difference between individual article landmarks if they're not labeled.

**Alternatives**

We could also potentially replace the `<article>` tags around features with `<div>` to satisfy the landmark issue, though this would remove the semantic meaning on each feature.

**Possible Implementation**

- Add `aria-label` to `article` tags in template-category.php, single.php and template-twocol.php, using the title of the category term and post title as the value, respectively
- Add `aria-labelledby` to `article` tags in each feature layout in includes/features.php; then add a unique ID to each `.feature-title` to reference in the `aria-labelledby` value, e.g. `<h2 class="feature-title" id="feature-<?php echo $post->ID; ?>">`

**Additional context**

https://github.com/UCF/Today-Child-Theme/search?utf8=%E2%9C%93&q=%3Carticle&type=
